src/confluent_kafka/src/Admin.c

```diff
@@ -329,6 +329,98 @@ static int Admin_set_replica_assignment (const char *forApi, void *c_obj,
         return 1;
 }
 
+
+static int
+Admin_incremental_config_to_c(PyObject *dict, void *c_obj){
+        Py_ssize_t pos = 0;
+        PyObject *ko, *vo;
+        PyObject *ks = NULL, *ks8 = NULL;
+        PyObject *local_ks = NULL, *local_ks8 = NULL;
+        PyObject *local_vs = NULL, *local_vs8 = NULL;
+
+        while(PyDict_Next(dict,&pos,&ko,&vo)){
+                const char *k;
+                const char *v = NULL;
+                int op = -1;
+                rd_kafka_error_t *error;
+                Py_ssize_t iter = 0;
+                PyObject *local_ko,*local_vo;
+
+                if (!(ks = cfl_PyObject_Unistr(ko))) {
+                        PyErr_Format(PyExc_ValueError,
+                                     "expected config name to be unicode "
+                                     "string");
+                       goto err;
+                }
+                k = cfl_PyUnistr_AsUTF8(ks, &ks8);
```

Comment:
I think this logic of converting a PyObject * string to char * should be moved to a function and reused. This makes the code unreadable in general. General comment. Not necessary to implement right now.